.github/workflows/publish-cloud.yml

```diff
@@ -0,0 +1,146 @@
+name: Publish Cloud
+# Required secrets for this workflow: MAINTAIN_USERNAME, MAINTAIN_EMAIL, SLACK_TOKEN, SLACK_CHANNELS, MAINTAIN_TOKEN, REMOTE_PUBLISH_REPO
+
+on:
+  repository_dispatch:
+    types: [deploy-cloud]
+  workflow_dispatch:
+    inputs:
+      version:
+        type: string
+        description: 'Release branch version'
+        required: false
+      pre_release:
+        type: boolean
+        description: 'Is this a pre-release?'
+        required: false
+  workflow_call:
+    inputs:
+      version:
+        type: string
+        description: 'Release branch version'
+        required: false
+      pre_release:
+        type: boolean
+        description: 'Is this a pre-release?'
+        required: false
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-20.04
+    outputs:
+      prev_version: ${{ steps.bump_version_step.outputs.prev_version }}
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.MAINTAIN_TOKEN }}
+          repository: ${{ github.event.client_payload.repository}}
+      - name: get configuration
+        id: configuration
+        uses: antifree/json-to-variables@v1.0.1
+        with: 
+            filename: './.github/workflows/config.json'
+            prefix: config
+      - name: Test permissions
+        uses: maork-elementor/elementor-editor-github-actions/permissions@master
```

Comment:
Replace all actions to local relative path

Reply:
cannot be done.